release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+:_content-type: ASSEMBLY
+[id="release-notes-371"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.71
+include::modules/common-attributes.adoc[]
+:context: release-notes-371
+
+toc::[]
+
+*Release date:* 25 July 2022
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime. It deploys in your infrastructure and integrates with your DevOps tooling and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+[id="about-this-release-371"]
+
+{product-title-short} 3.71 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+[id="new-features-371"]
+== New features
+
+[id="dashboard-updates"]
+=== Dashboard updates
+
+{product-title-short} has revised its dashboard and added new widgets to allow you to quickly review information about your system. You can view information for all clusters and namespaces or select individual clusters and namespaces for the display. Currently, settings that you select for the widget are saved in browser cookies, and are not available after rebooting your system. For more information on configuring the new dashboard widgets, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.71/html/operating/view-dashboard[Viewing the Dashboard]. 
```

Comment:
Yeah, the links don't work yet (since docs won't be published until 7/25). I wasn't sure about dashboard vs Dashboard but I looked at OCP and it's not capitalized in [those docs](https://docs.openshift.com/container-platform/4.10/web_console/using-dashboard-to-get-cluster-information.html) so I didn't capitalize to be consistent. I'm willing to reconsider if you think it should be though....

Reply:
@kcarmichael08 Do you want to capitalize the first instance of "dashboard" right after {product-title-short}?

Reply:
Yes, per our slack negotiation :)